svs/templates/svs/filter_result/row_sv.html

```diff
@@ -182,6 +182,15 @@
         <a class="dropdown-item" href="http://gnomad.broadinstitute.org/region/{{ first_entry.chromosome }}-{{ first_entry.start }}-{{ first_entry.end }}" target="_blank">
           Locus @gnomAD
         </a>
+        {% if first_entry.sv_type == "DEL" or first_entry.sv_type == "DUP" %}
+          <a class="dropdown-item" href="{% linkout_base_url "franklin_sv" first_entry.release %}{{ first_entry.chromosome }}-{{ first_entry.start }}-{{ first_entry.end }}-{{ first_entry.sv_type }}" target="_blank">
```

Comment:
`-{{ entry.alternative }}` should be suffixed by `-hg38` for GRCh38 or no suffix for hg19/GRCh37